### DISCUSSION TITLE
Deploy AWS ECS Fargate + EFS

### DISCUSSION THREAD

**Discussion by alvarolorentedev** [Budibase/budibase#11181] [Help]
I Have managed to deploy the Image of budibase to AWS Fargate with a persistent EFS volume. Nevertheless, I am having issues to connect to an MS SQL database as it gives me an error of timeout.
```json
{
    "connected": false,
    "error": "Failed to connect to ____ in 15000ms"
}
```

This only happens when running from fargate and not from my local machine. I have checked and there is network connectivity in between the cluster and the DB.

My suspects at this point are:
- Write permissions of the EFS volume. Is budibase using a specific uid?
- Size of the task (I have currently applied 2048 CPU & 4096 of Memory, I believe this is quite a bit more than recommended)

**Comment by ConorWebb96** [comment]
Hey @kanekotic,

As far as I know Budibase is not running under a specific uid.

I'd recommend that you verify that the security group attached to their Fargate's task allows outbound connections to the port that your MS SQL server is listening on (usually 1433), and likewise, the security group of your MS SQL server allows inbound connections from the security group attached to their Fargate task. 

Can you check your logs and see if there are anymore additional details relating to your error?

I hope this helps.

**Comment by andygrillo** [comment]
@kanekotic Hi, please can you share your task definition ? I am about to run Budibase on ECS and would be a great help. thanks

**Reply by alvarolorentedev** [reply]
sorry I am late to respond had to many notifications, did you manage or if you want i can still share on monday

**Comment by illoyd** [comment]
Hi @kanekotic, I would also appreciate seeing your CDK or CFN definition. I simply cannot get the Budibase service to start; the service continuously fails with no logs indicating an issue.

**Comment by regi4life** [comment]
Hi @kanekotic I would like to do the same thing : Deploying budibase on Amazon ECS ? Would you mind please to share your ECS task definition ?
